.github/workflows/mvn-branch-full-ci.yaml

```diff
@@ -0,0 +1,107 @@
+name: Build Branch (Full)
```

Comment:
Technically, when you add an commit to the feature branch, the build would be triggered, correct ?